lib/github-oauth-prompt.js

```diff
@@ -210,6 +209,11 @@ function getOptions (userOptions) {
         throw new Error('Option name must be a non-empty string');
     }
 
+    // Option: host.
+    if (options.host !== undefined && !isNonEmptyString(options.host)) {
```

Comment:
You don't need to check `undefined` because `isNonEmptyString` accounts for that.